.github/workflows/test_everest_macos.yml

```diff
@@ -7,9 +7,6 @@ on:
     branches:
       - main
     tags: "*"
-    paths:
```

Comment:
This seems to undo the prev commit of only running Mac tests on push to everest paths, intentional?

Reply:
It was intenetional, because it already just runs on tags, from below:

```yaml
if: github.ref_type == 'tag' # only build for mac when tags
```